backend/.run/Run services.run.xml

```diff
@@ -0,0 +1,11 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run services" type="docker-deploy" factoryName="docker-compose.yml" server-name="Docker">
```

Comment:
To jakieś specyficzne ustawienia do łatwego deployowania w docker compose?

Reply:
to jest konfiguracja do pycharma, żeby uruchomić docker-compose z postgresem jednym kliknięcie
![Screenshot from 2022-05-07 17-18-47](https://user-images.githubusercontent.com/44374201/167260612-151059ab-215a-4aee-b4a4-302bb388456c.png)